test/gtest/ucp/test_ucp_ep.cc

```diff
@@ -62,4 +61,37 @@ UCS_TEST_P(test_ucp_ep, ucp_query_ep)
                     estimated_time_1000 - estimated_time_0);
 }
 
+UCS_TEST_P(test_ucp_ep, ucp_query_transport)
+{
+    ucp_ep_h ep;
+    ucs_status_t status;
+    int i;
+    ucp_ep_attr_t ep_attrs;
+
+    ep                              = sender().ep();
+    ep_attrs.field_mask             = UCP_EP_ATTR_FIELD_TRANSPORTS;
+    ep_attrs.transports.entries     = new ucp_transport_entry_t[10];
+    ep_attrs.transports.num_entries = 100;
+    ep_attrs.transports.entry_size  = sizeof(ucp_transport_entry_t);
+    status                          = ucp_ep_query(ep, &ep_attrs);
```

Comment:
```

ucs_status_t status = ucp_ep_query(ep, &ep_attrs);
ASSERT_UCS_IS_OK(status);

```

Reply:
Where is ASSERT_UCS_IS_OK defined? I cannot find it anywhere in UCX source, even with a recursive grep of all files from the top of the directory structure?

Reply:
Looks like it should be ASSERT_UCS_OK

Reply:
Sorry, it's ASSERT_UCS_OK